test/balancerd/mzcompose.py

```diff
@@ -186,6 +190,126 @@ def workflow_http(c: Composition) -> None:
     # because there's no CNAME. Does docker-compose support this somehow?
 
 
+def workflow_http_tcp_proxy(c: Composition) -> None:
+    """Test http endpoint with tcp proxy"""
```

Comment:
This is a really nice test! I think the name might be from an early iteration of it, though?

```suggestion
def workflow_ip_forwarding(c: Composition) -> None:
    """Test that forwarding the client IP through the balancer works over both HTTP and SQL."""
```